Kernel/build-root-filesystem.sh

```diff
@@ -74,6 +74,12 @@ mknod mnt/dev/hdd b 4 1
 for hd in a b c d; do
     chmod 600 mnt/dev/hd$hd
 done
+mknod mnt/dev/sda b 8 0
+mknod mnt/dev/sdb b 8 16
```

Comment:
For now it might be OK to add those files, but in the future we should use a `devtmpfs` filesystem for it.
Maybe we can use the current `hda`, `hdb`... files?